ThreeDSecure/src/main/java/com/braintreepayments/api/ThreeDSecureRequest.java

```diff
@@ -381,7 +386,7 @@ public String build(String dfReferenceId) {
             base.put("additional_info", additionalInfo);
             base.putOpt("account_type", accountType);
 
-            if (cardAddChallengeRequested) {
+            if (cardAddChallengeRequested != null) {
```

Comment:
@scannillo this was the one change we had to add to make that possible, otherwise the parameter would always be set whether `true` or `false`. Now when this values `null`, the request is the same as the current version, and the behavior is up to the gateway.